src/corespotlight.cs

```diff
@@ -94,9 +94,14 @@ interface CSSearchableIndex {
 		// from interface CSExternalProvider (CSSearchableIndex)
 
 		[Async (ResultTypeName="CSSearchableIndexBundleDataResult")]
-		[NoTV, iOS (16,0), Mac (13,0), MacCatalyst (16,0)]
+		[iOS (16,0), Mac (13,0), MacCatalyst (16,0)]
```

Comment:
Why this change? I don't see it in the todo files?

Reply:
The class already has a NoTV